src/Microsoft.Win32.Registry/src/Microsoft/Win32/Registry.cs

```diff
@@ -14,7 +14,6 @@ namespace Microsoft.Win32
      * @security(checkClassLinking=on)
      */
     //This class contains only static members and does not need to be serializable.
-    [ComVisible(true)]
```

Comment:
Spoke with @davkean offline; these are already stripped from the contracts, so no issue there.